packages/cli/src/api/compile.test.ts

```diff
@@ -67,14 +68,165 @@ describe("compile", () => {
     )
   })
 
+  it("should compile multiple plurals", () => {
+    expect(
+      getSource(
+        "{bcount, plural, one {boy} other {# boys}} {gcount, plural, one {girl} other {# girls}}"
+      )
+    ).toEqual(
+      '[["bcount","plural",{one:"boy",other:["#"," boys"]}]," ",["gcount","plural",{one:"girl",other:["#"," girls"]}]]'
+    )
+  })
+
   it("should report failed message on error", () => {
     expect(() =>
       getSource("{value, plural, one {Book} other {Books")
     ).toThrowErrorMatchingSnapshot()
   })
+
+  describe("with pseudo-localization", () => {
+    const getPSource = (message: string) => getSource(message, true)
+
+    it("should pseudolocalize strings", () => {
+      expect(getPSource("Martin Černý")).toEqual('"Màŕţĩń Čēŕńý"')
+    })
+
+    it("should pseudolocalize escaping syntax characters", () => {
+      // TODO: should this turn into pseudoLocale string?
+      expect(getSource("'{name}'", true)).toEqual('"{name}"')
+      // expect(getSource("'{name}'", true)).toEqual('"{ńàmē}"')
+    })
+
+    it("should not pseudolocalize arguments", () => {
+      expect(getPSource("{name}")).toEqual('[["name"]]')
+      expect(getPSource("B4 {name} A4")).toEqual('["ß4 ",["name"]," À4"]')
+    })
+
+    it("should not pseudolocalize arguments nor formats", () => {
+      expect(getPSource("{name, number}")).toEqual('[["name","number"]]')
+      expect(getPSource("{name, number, percent}")).toEqual(
+        '[["name","number","percent"]]'
+      )
+    })
+
+    it("should not pseudolocalize HTML tags", () => {
+      expect(getPSource('Martin <span id="spanId">Černý</span>')).toEqual(
+        '"Màŕţĩń <span id=\\"spanId\\">Čēŕńý</span>"'
+      )
```

Comment:
1. Maybe add a test case with both `'` and `"` in it.
2. Because there's escaping in the string you send to `toEqual`, but also in the JSON it represents, i'd use a template string instead to prevent the double escapes, and possibly anywhere there's any type of quote-mark in a string here

```suggestion
      expect(getPSource(`Martin <span id="spanId">Černý</span>`)).toEqual(
        `"Màŕţĩń <span id=\"spanId\">Čēŕńý</span>"`
      )
```